components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -271,7 +294,7 @@ lnd/clm2/surfdata_map/surfdata_1x1_brazil_simyr1850_c140610.nc</fsurdat>
 
 
 <fsurdat hgrid="ne30np4"      sim_year="1850" use_crop=".false." >
-lnd/clm2/surfdata_map/surfdata_ne30np4_simyr1850_c180306.nc</fsurdat>
+lnd/clm2/surfdata_map/surfdata_ne30np4_simyr1850_2015_c171018.nc</fsurdat>
```

Comment:
Why is the timestamp going backwards?  From 180306 to 171018 ?

Reply:
Because all supported 1850 compsets were using the older file. (e.g. https://github.com/E3SM-Project/E3SM/pull/3333/files#diff-7ece8b6f482388c76e3a4cab97c792eeL34)

Reply:
The c171018 file is the surfdata file used in the CMIP6 compsets.   I don't know where the c180306 file came from, or why it is a default.

Reply:
Where did the c180306 file come from?
What was it for?